### ISSUE TITLE
[Bug] webinstall.dev releases (json|tab) not showing Go past 1.19

### ISSUE THREAD

**Issue by HacDan** [webinstall/webi-installers#643]
## What were you trying to install (or what else went wrong)?

N/A

## What exactly did you do?

I went to https://webinstall.dev/api/releases/golang.json?pretty=true to see if 1.21 was showing in releases yet for Go.

## What went wrong?

I noticed that no version past 1.19Beta 1 was showing
## Which OS did you try on?

- [x] Mac, although I don't think it matters
- [ ] Windows
- [ ] Linux

## What type of computer (i.e. laptop, desktop, Raspberry Pi)?

- [x] Laptop or Desktop (amd64 / x86-64) (arm64)
- [ ] Raspberry Pi (ARM-64)
- [ ] Other (please specify, if you know)

**Comment by coolaj86** [comment]
I think there's just too many releases and the page is being truncated.

If you add some qualifiers (OS, CPU, Channel), then it'll show the more recent versions:
<https://webinstall.dev/api/releases/golang.json?pretty=true&channel=stable>

This weirdness of the default sort in this case is possibly specific to Go because it doesn't use semver naming and is converted later in the pipeline.

**Comment by coolaj86** [comment]
You can also see them if you increase the limit:
https://webinstall.dev/api/releases/golang.json?pretty=true&limit=10000

I'm going to close this as a bug, but I'd like to keep the conversation going.

To add paging to the default list would break existing implementations. But do you have any other ideas?

We could change the link to include:
- include the `limit=<default>` parameter (I think it's 1000)
- include `channel=stable`

We could figure out how to change the sorting earlier in the pipeline

We could add a `page=1` parameter that changes the output to include pagination information (or perhaps better, a different URL entirely to avoid type issues).

**Comment by HacDan** [comment]
I'm not sure Pagination is needed, but I think some sane defaults would be helpful when checking the "feed" so to speak. 

I'd say defaulting `channel=stable` would make sense. I think in think in this case, the sorting was what I was missing. I noticed 1.19Beta1 and assumed 1.21 would be at the top and moved on without checking further down the feed. 

I did notice when changing to `channel=stable` the sorting resolves. Presumably because whatever is sorting can now successfully sort the number and the addition of alpha characters throws things off.
